File: Modul 4/NO 4/PRAK304-2210817310016-Riyo Aurora Guison.py
a=int(input(""))
if(a>=100):
    print("Anda Menginput Melebihi Limit Bialangan")
elif(a>=20):
    print("Puluhan")
elif(a>10 and a<20):
    print("Belasan")
elif(a>0 and a<10):
    print("Satuan")
elif(a==0):
    print("Nol")

File: Modul 4/NO 4/PRAK304-2210817310016-Riyo Aurora Guison.c
#include<stdio.h>
int main()
{
int a;
printf("");
scanf("%d",&a);
     if(a>=100){
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    else if(a>=20){
        printf("Puluhan\n");
    }
    else if(a>10 && a<20){
        printf("Belasan\n");
    }
    else if(a>0 && a<10){
        printf("Satuan\n");
    }
    else if(a==0){
        printf("Nol\n");
    }
}

